src/components/Form/Form.js

```diff
@@ -47,7 +96,32 @@ const form = () => {
     });
   };
 
-  const panelType = quizForm[formState.step].typePanel || '';
+  let panelType = quizForm[formState.step].typePanel || '';
+
+  const calcAnswersTOTAL = updatedForm => {
```

Comment:
I would try to get this information from the store, as you already have to update the state per answered question, it would be easy to have the value stored there

Reply:
I found it, from performance point of view to have a function to calculate all the answered questions and I called once. 

`Form.js` is a Functional Component and is does not have proper State

Reply:
In general "performance-reasons" or rather performance optimizations should be tackled when measurable / testable issues are discovered in an early version - as long as there is no issue I would always follow the best approach for maintainability and or readability.